test/unit/content-info.test.js

```diff
@@ -23,6 +24,23 @@ describe('contentInfo module', function () {
         });
     });
 
+    // eslint-disable-next-line max-len
```

Comment:
not required.

Reply:
it is now. after applying condition for skipping the test for browser.